setup.cfg

```diff
@@ -12,14 +12,19 @@ long_description = file: README.rst, CHANGELOG.rst, LICENSE
 url              = 'https://github.com/riptideio/pymodbus/'
 classifiers      = 
     Development Status :: 4 - Beta
-    Programming Language :: Python :: 3.8 :: 3.9 :: 3.10
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
     Environment :: Console
-    Framework :: Twisted :: Tornado :: Asyncio
-    Intended Audience :: Developers :: Modbus testers
+    Framework :: Asyncio
+    Intended Audience :: Developers
     License :: OSI Approved :: BSD License
-    Operating System :: OS Independent :: Linux :: Windows :: Mac OS
-    Programming Language :: Python :: 3.8
-    Topic :: System :: Networking :: Utilities :: SCADA :: Modbus
+    Operating System :: OS Independent :: Linux
```

Comment:
There is no `Operating System :: OS Independent :: Linux` , we can use `Operating System :: POSIX :: Linux
`

Reply:
This is the working diff
```diff --git a/setup.cfg b/setup.cfg
index 6d4e6433..94c61680 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -9,17 +9,26 @@ platforms        = 'Linux', 'Mac OS X', 'Win'
 description      = "A fully featured modbus protocol stack in python"
 keywords         = 'modbus, asyncio, scada, client, server, simulator'
 long_description = file: README.rst, CHANGELOG.rst, LICENSE
-url              = 'https://github.com/riptideio/pymodbus/'
-classifiers      =
+url              = https://github.com/riptideio/pymodbus/
+classifiers      =
     Development Status :: 4 - Beta
-    Programming Language :: Python :: 3.8 :: 3.9 :: 3.10
     Environment :: Console
-    Framework :: Twisted :: Tornado :: Asyncio
-    Intended Audience :: Developers :: Modbus testers
+    Framework :: Twisted
+    Framework :: AsyncIO
+    Intended Audience :: Developers
     License :: OSI Approved :: BSD License
-    Operating System :: OS Independent :: Linux :: Windows :: Mac OS
+    Operating System :: POSIX :: Linux
+    Operating System :: Unix
+    Operating System :: MacOS :: MacOS X
+    Operating System :: OS Independent
+    Operating System :: Microsoft
     Programming Language :: Python :: 3.8
-    Topic :: System :: Networking :: Utilities :: SCADA :: Modbus
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
+    Topic :: System :: Networking
+    Topic :: Utilities
 project_urls     =
     Source Code  = https://github.com/riptideio/pymodbus
     Bug Reports  = https://github.com/riptideio/pymodbus/issues
(END)
```

Reply:
OK for me. But should we really announce 3.11 and 3.12 ? I have not tested 3.11 and know there are problems with 3.12

Reply:
Will you commit the changes directly ?

Reply:
Yes, I will do it once I figure out the issue with the `long_description`

Reply:
Thanks, I have update README.rst to reflect that we now are at dev4.

Reply:
Cool , I have the release published to Pypi https://pypi.org/project/pymodbus/3.0.0.dev4/

Reply:
Super, more testing for me 😃